app/assets/javascripts/vocab_bar.js

```diff
@@ -24,8 +24,8 @@ $(document).ready(function() {
       if (barBottom > footerTop) {
         if ($('.search-results__container').length > 0) {
           bar.css({height: $('.search-results__container').innerHeight()});
-        } else if ($('.search-result-container').length > 0) {
```

Comment:
`.search-result-container` has a wider clearfix issue which results in it having no height. Switched to vocab_sheet_background which a) works and b) seems like a better element to target here.